io/post_fv3.F90

```diff
@@ -2138,164 +2335,641 @@ subroutine set_postvars_fv3(wrt_int_state,grid_id,mype,mpicomp)
               enddo
             endif
 
-            ! inst  cloud top pressure
-            if(trim(fieldname)=='prescnvclt') then
```

Comment:
@WenMeng-NOAA So we don't output inst/time averaged cloud top/mid/bottom pressure any more?

Reply:
@junwang-noaa These variables are still kept in the code. There are a lot of new variables inserted before them.